bastion/gui_qt.py

```
@@ -967,6 +1084,24 @@ def disable_ufw(self):
             show_notification(self, "Error", f"Failed to disable UFW: {e}")
         self.refresh_ui()
 
+    def start_tray_icon(self):
+        """Start the system tray icon (bastion-gui)"""
+        from .notification import show_notification
```

Ruff (0.14.10)

Redefinition of unused `show_notification` from line 23

Remove definition: `show_notification`

(F811)